dependencies.yaml

```diff
@@ -16,9 +16,8 @@ dependencies:
         match: GO_VERSION
       - path: nix/derivation.nix
         match: buildGo124Module
-      # TODO: move go.mod to 1.24 once we have full prow CI support
-      # - path: go.mod
-      #   match: go
+      - path: go.mod
```

Comment:
Did we confirm if Prow supports Go 1.24?

Reply:
Tests succeed, I assume it has 1.24 now.

Reply:
Openshift did not seem to have go 1.24 work defined yet. It may be worth following up with golang team to see when go 1.24 will be available for openshift/prow.

Reply:
We need go 1.24 support in any case because k/k already switched over, which is then also a requirement for https://github.com/cri-o/cri-o/pull/9050. Not exactly sure why we should block this PR on downstream, though.